### ISSUE TITLE
How to preprocess LIDC-IDRI ? how to split training set and test set with LIDC-IDRI ?

### ISSUE THREAD

**Issue by tricoffee** [wentaozhu/DeepLung#82]
How to preprocess LIDC-IDRI ? how to split training set and test set with LIDC-IDRI ?

**Comment by wentaozhu** [comment]
You may use 10-fold cross validation which randomly generates 10 folds and selects one for validation for 10 times.
